baselines/webworker.generated.d.ts

```diff
@@ -344,8 +349,8 @@ interface IDBIndex {
     count(key?: any): IDBRequest;
     get(key: any): IDBRequest;
     getKey(key: any): IDBRequest;
-    openCursor(range?: IDBKeyRange, direction?: string): IDBRequest;
-    openKeyCursor(range?: IDBKeyRange, direction?: string): IDBRequest;
+    openCursor(range?: any, direction?: string): IDBRequest;
+    openKeyCursor(range?: any, direction?: string): IDBRequest;
```

Comment:
This is too lax, as I mentioned in https://github.com/Microsoft/TypeScript/issues/7218#issuecomment-188668347